### DISCUSSION TITLE
Connection to Localhost is refused

### DISCUSSION THREAD

**Discussion by gensart-ai** [thunderclient/thunder-client-support#1503] [Q&A] [answered]
Any wonder why Thunder can't hit a request to localhost ?

![image](https://github.com/rangav/thunder-client-support/assets/58139648/7245b961-6a94-49fa-a7fc-e43b71dda73f)

Is there any settings that I missed ? Thank you

**Comment by rangav** [comment, marked as answer]
see solution here
https://github.com/rangav/thunder-client-support/blob/master/docs/common-issues.md

**Reply by gensart-ai** [reply]
Thanks for the attention,

By your solution, it works on mine when using `[::1]` and `[::]` as `localhost` replacement in my endpoint.

**Reply by gensart-ai** [reply]
Oh also, for future readers, I set the Http Library to `axios` in Thunder client extension settings.

![image](https://github.com/rangav/thunder-client-support/assets/58139648/8b118136-33fc-4a84-aac0-8e373b6b0c34)
